### PR TITLE
Docs : add gs scheme support to docs

### PR DESCRIPTION
**Pull request by Nemric**
It seems that gs scheme is supported since 1.2/3.2 version of butane/ignition but docs doesn't mention it

### CODE REVIEW

File: docs/config-fcos-v1_2.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The Fedora CoreOS configuration is a YAML document conforming to the following s
 * **_ignition_** (object): metadata about the configuration itself.
   * **_config_** (objects): options related to the configuration.
     * **_merge_** (list of objects): a list of the configs to be merged to the current config.
-      * **_source_** (string): the URL of the config. Supported schemes are `http`, `https`, `s3`, `tftp`, and [`data`][rfc2397]. Note: When using `http`, it is advisable to use the verification option to ensure the contents haven't been modified. Mutually exclusive with `inline` and `local`.
+      * **_source_** (string): the URL of the config. Supported schemes are `http`, `https`, `s3`, `gs`, `tftp` and [`data`][rfc2397]. Note: When using `http`, it is advisable to use the verification option to ensure the contents haven't been modified. Mutually exclusive with `inline` and `local`.
```
Comment (bgilbert) [inline docs/config-fcos-v1_2.md:16]: Let's retain the serial comma in all these lists.

**Review by bgilbert — COMMENTED** [review]
Thanks for the PR!  For completeness, let's update the `rhcos` 0.1 spec too, even though it's deprecated.  The `openshift` specs can be left as is; see #299.

Also one typographic nit.

File: docs/config-fcos-v1_2.md (hunk 1 above)
Comment (Nemric) [reply docs/config-fcos-v1_2.md]: Done in all files :)

**Comment by bgilbert** [comment]
Thanks for the comma fix.  If you could also update the `rhcos` 0.1 spec and squash your commits, this should be good to go. :+1:

**Comment by bgilbert** [comment]
Accidentally closed; replaced with #305.

**Pull request closed without merging**